src/main/scala/lockbook/dev/SettingsUi.scala

```diff
@@ -0,0 +1,66 @@
+package lockbook.dev
+
+import java.util.Optional
+
+import javafx.collections.FXCollections
+import javafx.geometry.Insets
+import javafx.scene.control.ButtonBar.ButtonData
+import javafx.scene.control._
+import javafx.scene.layout.GridPane
+
+class SettingsUi(settingsHelper: SettingsHelper, fileHelper: FileHelper) {
+
+  def getView(): Unit = { //use graphbox
+
+    val dialog: Dialog[Settings] = new Dialog[Settings]()
+
+    dialog.getDialogPane.getButtonTypes.addAll(
+      ButtonType.APPLY,
+      ButtonType.CANCEL
+    )
+
+    val gridPane = new GridPane
+
+    getViewHelper(dialog, gridPane)
+
+    val result: Optional[Settings] = dialog.showAndWait()
+    if (result.isPresent) {
+      SettingsHelper.constructJson(result.get(), fileHelper)
+    }
+  }
+
+  def getViewHelper(dialog: Dialog[Settings], gridPane: GridPane): Unit = {
+    val stylesBox       = new ComboBox[String](FXCollections.observableArrayList(Light.fileName))
+    val autoLockTimeBox = new ComboBox[Int](FXCollections.observableArrayList(5, 2))
```

Comment:
So there's a bit more complexity to model here. 

There should be a a way to disable the `AutoLock` feature all together as well as a way to specify any value. So the UI should expose something like a checkbox, if it's checked it enables the text field for the setting, and if it's not checked it disables it.

Behind the scenes this should be something like `Option[Int]` or preferably `Option[Duraction]`. Where `None` implies the feature is disabled. Would be okay with representing this as an `Either[Disabled, Duration]` as well. 

But the 2 pieces of feedback here are: should be able to turn this off, and should be able to set any arbitrary value.